charts/ditto/requirements.yaml

```
@@ -11,6 +11,6 @@
 ---
 dependencies:
   - name: mongodb
-    repository: https://kubernetes-charts.storage.googleapis.com
+    repository: https://charts.bitnami.com/bitnami
     version: 7.6.3
```

```suggestion
    version: ~7.14.7
```
Should we use the same version as Hono defines?

---

Well, I am glad to align the versions. However, you will need to tell me if Ditto will work with it ...

---

Yes, it will - this references the latest MongoDB 4.2 which Ditto 1.5.0 was explicitly tested against.

---

done